packages/flutter_tools/lib/src/application_package.dart

```diff
@@ -392,6 +392,9 @@ class ApplicationPackageStore {
   AndroidApk android;
   IOSApp iOS;
   FuchsiaApp fuchsia;
+  LinuxApp linux;
+  MacOSApp macOS;
+  WindowsApp windows;
 
   Future<ApplicationPackage> getPackageForPlatform(TargetPlatform platform) async {
```

Comment:
We could a simple smoke test here by asserting that the returned value for a particular platform is non-null